src/amber/server/configuration.cr

```diff
@@ -44,6 +44,10 @@ module Amber
         settings.router
       end
 
+      def self.handler
```

Comment:
Convenience method to write `Amber::Server.handler` instead `Amber::Server.settings.handler`

Reply:
I don't think the base handler should be included in settings to begin with. It's not a setting as the entire routing system depends on it.

Reply:
Same with pubsub_adapter and router. WDYT? It's not what come to mind for me when I think of settings.

Reply:
Agreed. See https://github.com/amberframework/amber/pull/386